src/libraries/Common/src/System/IO/MemoryMappedFileInternal.Windows.cs

```diff
@@ -2,43 +2,50 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.Win32.SafeHandles;
```

Comment:
MemoryMappedFile.Windows.cs should stay, and that file's methods should call the MemoryMappedFileInternal ones, otherwise the Unix build fails.